shadow/main/Main.hs

```diff
@@ -0,0 +1,83 @@
+{- © 2019 Serokell <hi@serokell.io>
+ - © 2019Lars Jellema <lars.jellema@gmail.com>
+ -
+ - SPDX-License-Identifier: MPL-2.0
+ -}
+
+{-# LANGUAGE DeriveDataTypeable #-}
+
+module Main where
+
+import Control.Concurrent
+import Control.Concurrent.ParallelIO.Local
+import Data.Either
+import qualified Data.Text.IO as TextIO
+import GHC.Conc (numCapabilities)
+import System.Console.CmdArgs
+import System.Exit
+import System.IO
+import System.Posix.Process (exitImmediately)
+import System.Posix.Signals
```

Comment:
You have to specify what do you import [explicitly](https://github.com/serokell/serokell-util/blob/master/serokell-style.md#imports) (unless you define your own style-guide).

Reply:
Btw, our default style guide is [here](https://github.com/serokell/style).